presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -22,6 +22,11 @@
 import org.testcontainers.shaded.com.google.common.collect.ImmutableList;
```

Comment:
Can you please explain (in the commit message?) what was the issue that you are solving and how it is going to work now?